hil/cli.py

```diff
@@ -288,6 +288,17 @@ def serve_networks():
         sleep(sleep_time)
 
 
+@cmd
+def list_user():
+    """List all users when the database authentication is active.
+
+    Admin priveledge required.
```

Comment:
*privilege

Reply:
i copied and pasted from a docstring below lol. Found two typos at one time ;)